src/assignments/benson/lesson_3_answers_test.clj

```diff
@@ -0,0 +1,15 @@
+(ns lesson-3-answers-test
+  (:require [assignments.benson.lesson-3-answers :refer [best-hand] :as sut]
+           [clojure.test :refer (deftest is are)]))
+
+(deftest best-hand-test
```

Comment:
Did you consider which of your helper functions might need testing?  Which ones do you think are less than obvious and would provide a nice extra documentation?  :-)